### PR TITLE
Next set of upgrades to node.

### PR DESCRIPTION
**Pull request by jnaviask**

### CODE REVIEW

File: node/runtime/src/lib.rs [hunk 1]
```diff
@@ -932,9 +955,27 @@ static EVM_CONFIG: EvmConfig = EvmConfig {
 	has_ext_code_hash: true,
 	estimate: false,
 };
+/// Current (safe) approximation of the gas/s consumption considering
+/// EVM execution over compiled WASM.
+pub const GAS_PER_SECOND: u64 = 8_000_000;
```
Comment (drewstone) [inline node/runtime/src/lib.rs:960]: This might be high come to think of it, but we should benchmark accordingly.

File: node/runtime/src/lib.rs [hunk 2]
```diff
@@ -932,9 +955,27 @@ static EVM_CONFIG: EvmConfig = EvmConfig {
...
+		Weight::try_from(gas.saturating_mul(WEIGHT_PER_GAS)).unwrap_or(Weight::MAX)
+	}
+	fn weight_to_gas(weight: Weight) -> u64 {
+		weight.wrapping_div(WEIGHT_PER_GAS)
+	}
+}
 
 impl pallet_evm::Config for Runtime {
-	type FeeCalculator = FixedGasPrice;
+	type FeeCalculator = (); // TODO: permit validators to set minimum gas prices
```
Comment (drewstone) [inline node/runtime/src/lib.rs:978]: I think we should still use the FixedGasPrice as this is at the runtime level. The validators setting it is at the client level, no?

**Review by drewstone — APPROVED** [review]
Looks good some additional ideas for tests when we start our exported state tests:
1. Checking all module indices are preserved
2. Checking treasury amounts continue to be minted from treasury reward

File: node/runtime/src/lib.rs (hunk 2 above)
Comment (jnaviask) [reply node/runtime/src/lib.rs:978]: All this means is that there's no minimum gas price at the runtime level -- we could just add a struct that says "0" if we wanted to be explicit about it.

File: node/runtime/src/lib.rs (hunk 1 above)
Comment (jnaviask) [reply node/runtime/src/lib.rs:960]: Yeah definitely should get some benchmarks before deploying this.

**Pull request merged**